docs/usage.md

```diff
@@ -56,10 +56,6 @@ to the pytest.
     [default config](https://github.com/red-hat-storage/ocs-ci/tree/master/ocs_ci/framework/conf/default_config.yaml).
     This is the repeatable parameter and you can use it multiple times for
     adding multiple config files. (The last one overwrites previous config!)
-* `--cluster-conf` - with this configuration you can overwrite the default
```

Comment:
only --cluster-conf should change to --ocsici-conf, the rest of the help message should exist.

Reply:
This parameter is described few lines above:  https://github.com/red-hat-storage/ocs-ci/pull/1259/files#diff-4be2078a125fe364f7198675d51bc688L54

Reply:
Done